torchgeo/datasets/cropharvest.py

```diff
@@ -0,0 +1,673 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""CropHarvest datasets."""
+
+import glob
+import json
+import os
+from typing import Callable, Optional
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import torch
+from matplotlib.figure import Figure
+from torch import Tensor
+
+from .geo import NonGeoDataset
+from .utils import DatasetNotFoundError, download_url, extract_archive
+
+
+class CropHarvest(NonGeoDataset):
+    """CropHarvest dataset.
+
+    The `CropHarvest <https://github.com/nasaharvest/cropharvest>`_ datataset is a
+    classification dataset for crop classification.
+
+    Dataset features:
+
+    * single pixel timeseries with croptype labels
+    * 18 bands per image over 12 months
+
+    Dataset format:
+
+    * images are 12x18 ndarrays with 18 bands over 12 months
```

Comment:
```suggestion
    * arrays are 12x18 with 18 bands over 12 months
```
We use Torch Tensors anyway